test/gotosource.test.ts

```diff
@@ -161,12 +161,27 @@ describe('VerifyError', () => {
     });
 
 
+    it('stop tokenUtxo.scrypt#14 => require(Tx.checkPreimage(txPreimage))', () => {
```

Comment:
Also add a test for checkSig() failure

Reply:
already have  at 
unit test stop at p2pkh.scrypt#10